test/setup.spec.js

```diff
@@ -177,54 +177,54 @@ describe('setup', () => {
     )
   })
 
-  it('not warn doing toRef on props', async () => {
-    const Foo = {
-      props: {
-        obj: {
-          type: Object,
-          required: true,
-        },
-      },
-      setup(props) {
-        return () =>
-          h('div', null, [
-            h('span', toRefs(props.obj).bar.value),
-            h('span', toRefs(props.obj.nested).baz.value),
-          ])
-      },
-    }
-
-    let bar
-    let baz
-
-    const vm = new Vue({
-      template: `<div id="app"><Foo :obj="obj" /></div>`,
-      components: { Foo },
-      setup() {
-        bar = ref(3)
-        baz = ref(1)
-        return {
-          obj: {
-            bar,
-            nested: {
-              baz,
-            },
-          },
-        }
-      },
-    })
-    vm.$mount()
-
-    expect(warn).not.toHaveBeenCalled()
-    expect(vm.$el.textContent).toBe('31')
-
-    bar.value = 4
-    baz.value = 2
-
-    await vm.$nextTick()
-    expect(warn).not.toHaveBeenCalled()
-    expect(vm.$el.textContent).toBe('42')
-  })
+  // it('not warn doing toRef on props', async () => {
```

Comment:
this is caused because the props are marked as `raw`

Open for suggestions

Reply:
Guess we can mark the props to shallow reactive and change the test to only tests the top-level `toRefs`, just like you mentioned in the limitation part?

Reply:
Props are already reactive, only the nested ref are not
https://github.com/vuejs/composition-api/blob/030d02ca0f67782629dc602f2f441a8712f78bb7/test/setup.spec.js#L336-L371

Reply:
I think it's fine by leaving a note in the limitation for this.